src-js/main.js

```diff
@@ -87,14 +89,14 @@ function detectScrollbar( element ){
 }
 
 function renderVersionBlockPopover() {
-  var badges = document.querySelectorAll(".server-version-badge");
-
```

Comment:
can you rebase w/ master? this shouldn't happen.

Reply:
i think your branch is overwriting some of the changes i merged into master like the share button stuff.

Reply:
Fixed! Pulled in latest version of file.